### PR TITLE
Text edits and d3force schanges

### PR DESCRIPTION
**Pull request by cnell-usgs**
This uses a declarative approach to writing the scrolling text steps. Also tweaks d3force and restructures how that changes with scrolling. Beginnings of adding more points to the beeswarm on scroll, although has not been successful yet.

One thing to note here is that I've changed the way that the `step id` is assigned. The ids are now declared in `modelingText.js` and added to each div using the `v-for` directive. I don't think this is the issue and have likely modified flubber unintentionally in another way when trying to merge branches.

### CODE REVIEW

File: src/views/3-Modeling/Modeling.vue [hunk 1]
```diff
@@ -4,292 +4,224 @@
...
     >
       <div id="flubber-container">
-        <div id="flubber">
-          <svg
-            id="transform-svg-test"
-            xmlns="http://www.w3.org/2000/svg"
-            width="300px"
-            height="300px"
+        <svg
+          id="flubber"
```
Comment (hcorson-dosch-usgs) [inline src/views/3-Modeling/Modeling.vue]: For no good reason I'm actually still using the id `"transform-svg-test"` for this svg. And right now, to work w/ the css, it does have to be within a div with the id `"flubber"`

File: src/views/3-Modeling/Modeling.vue [hunk 2]
```diff
@@ -4,292 +4,224 @@
...
         <div
-          id="ANN"
+          :id="text.flubber_id"
           class="step"
-          data-step="0"
         >
-          <p v-html="text.paragraph2" />
-        </div>
-      </div>
-      <div class="step-container text-content">
```
Comment (hcorson-dosch-usgs) [inline src/views/3-Modeling/Modeling.vue:192]: Hmm so I think it is the removal of all of these `"step-container text-content"` divs that is tripping up my flubber code, b/c I had assigned ids to these that were then triggering the flubber animation. Are these now elsewhere?

File: src/views/3-Modeling/Modeling.vue (hunk 1 above)
Comment (cnell-usgs) [reply src/views/3-Modeling/Modeling.vue]: that fixed it 😌

File: src/views/3-Modeling/Modeling.vue (hunk 2 above)
Comment (cnell-usgs) [reply src/views/3-Modeling/Modeling.vue:192]: yeah those are now named in `modelingText.js`. That is basically everything relating to the text steps is defined there and this section loops through that and creates the same div structure for each step. I think this is working with the steps, I added an element to put them in there as `flubber_id`

**Review by hcorson-dosch-usgs — APPROVED** [review]

**Pull request merged**